src/style.css

```diff
@@ -2,0 +1,50 @@
+body {
+  margin: 0;
+}
+
+main {
+  width: 96px;
+  height: 116px;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars__star {
+  width: 16px;
+  height: 16px;
+  background-image: url('./images/star.svg');
```

Comment:
Avoid using the `*` selector to apply styles. It is more efficient to apply styles to specific classes or elements when necessary.